docs/user/azure/migrate.md

```diff
@@ -0,0 +1,121 @@
+## Migrating legacy Azure DNS zones
+
+"openshift-install migrate azure privatedns" is a command for migrating Azure legacy private DNS zones to next generation private zones
+
+## Usage
+
+Arguments:
+
+    # Show legacy zones that are eligible to be migrated
+    eligible
+
+    # migrate a zone
+    migrate
+
+        # The zone we want to migrate (required)
+        --zone=example.com
+
+        # The resource group to create the private zone in (required)
+        --new-resource-group=rg
+
+        # The resource group of the legacy zone (optional)
+        --old-resource-group=rg
+
+        # The virtual network to create the private zone in (optional)
+        --virtual-network=vnet
+
+        # The resource group the virtual network is in (optional)
+        # Required if specifying a virtual network
+        --virtual-network-resource-group=vrg
+
+        # Link the newly created private zone to the virtual network for DNS (optional)
+        --link=vnet
+
+Examples:
+
+    # This will show legacy zones that can be migrated to private zones
+    openshift-install migrate azure privatedns eligible
+
+    # This will migrate a legacy zone to a private zone
+    # Without specifying -link, a virtual network link from the new zone to the vnet will not be created
+    openshift-install migrate azure privatedns migrate --old-resource-group=rg --new-resource-group=rg --zone=example.com --virtual-network-resource-group=rg --virtual-network=myvnet --link
+
+## Workflow for migrating a legacy private zone to a new private zone
+- list zones eligible for migration or know which ones to migrate
+- gather details (resource groups, virtual networks, etc)
+- perform the migration
+
+## Example migration:
+
+We start off with the empty resource group "jhixson_dns_migration_rg"
+
+![azure 1](images/azure-migrate-dns-1.png)
+
+Next, We create a legacy private zone with mock records
+
+Verify the legacy zone was created:
+
+![azure 2](images/azure-migrate-dns-2.png)
+
+Verify the records have been created: 
+
+![azure 3](images/azure-migrate-dns-3.png)
+
+Verify this is in fact, a legacy zone:
+
+![azure 4](images/azure-migrate-dns-4.png)
+
+Now, list zones that are eligible for migrating:
+
+```
+[jhixson@redlap test]$ ./openshift-install migrate azure privatedns eligible
+INFO[0000] Credentials loaded from file "/home/jhixson/.azure/osServicePrincipal.json" 
+INFO[0002] legacy zone=testmigrate.installer.azure.devcluster.openshift.com resourceGroup=jhixson_dns_migrate_rg 
```

Comment:
why is this not using the logging format like the rest of the install commands? Can we fix this?

Reply:
esp `INFO[0002]`

Reply:
I've updated the doc to reflect the changes.